src/valour.js

```diff
@@ -121,6 +121,15 @@ class Valour {
     });
     this.runCallbacks(name);
   }
+
+  disposeForm(name) {
+    this.forms[name] = {};
+  }
+
+  removeValidation(formName, key) {
```

Comment:
In we're sticking with the "form" metaphor, maybe this should be named something like `removeField`.